<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>380</width>
         <height>56</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="first">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="second">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="third">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="first_range_box">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>170</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="second_range_box">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>190</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="count_clasters"/>
    </item>
    <item>
     <widget class="QTableWidget" name="table">
      <column>
       <property name="text">
        <string>индекс</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>площадь</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>периметр</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>компактность</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>удлиненность</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>номер кластера</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="enter">
      <property name="text">
       <string>enter</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
